docs/serving/samples/blue-green-deployment.md

```diff
@@ -243,8 +247,13 @@ Refresh the original route
 (http://blue-green-demo.default.YOUR_CUSTOM_DOMAIN.com) a few times to verify
 that no traffic is being routed to v1 of the app.
 
-We added a named route to v1 of the app, so you can now access it at
-http://v1.blue-green-demo.default.YOUR_CUSTOM_DOMAIN.com.
+We added a named route to v1 of the app, so you can now access it at the URL
+listed in the traffic block of status. To get the url you can execute the
```

Comment:
I would say something like "..listed in the traffic block of the status section" or "list in the status section of the route manifest". 

"traffic block of status" does not seem right.